frame/vesting/src/lib.rs

```diff
@@ -156,9 +186,15 @@ pub mod module {
 			to: AccountIdOf<T>,
 			asset: AssetIdOf<T>,
 			schedule: VestingScheduleOf<T>,
+			vesting_schedule_id: T::VestingScheduleId,
 		},
 		/// Claimed vesting. \[who, locked_amount\]
-		Claimed { who: AccountIdOf<T>, asset: AssetIdOf<T>, locked_amount: BalanceOf<T> },
+		Claimed {
+			who: AccountIdOf<T>,
+			asset: AssetIdOf<T>,
+			locked_amount: BalanceOf<T>,
+			vesting_schedule_id: Option<T::VestingScheduleId>,
```

Comment:
```suggestion
			who: AccountIdOf<T>,
			asset: AssetIdOf<T>,
			vesting_schedule_id: Option<T::VestingScheduleId>,
			locked_amount: BalanceOf<T>,
```

Reply:
Just a nit, but this order reads better since it goes identifier -> data (if that makes sense)

Reply:
✅